Tests/Common/Scheduling/DateRulesTests.cs

```diff
@@ -102,6 +103,23 @@ public void StartOfMonthNoSymbolMidMonthStart()
             Assert.AreEqual(11, count);
         }
 
```

Comment:
Think it would be nice to add a test for `SPY/BTCUSD/EURUSD/Fut_SPY_Feb19_2016` with no offset and assert the base start/end weekday directly